easybuild/framework/easyblock.py

```diff
@@ -188,15 +188,27 @@ def __init__(self, ec):
             self.log.warning("Group spec '%s' is overriding config group '%s'." % (self.cfg['group'], group_name))
             group_name = self.cfg['group']
 
+        # group
         self.group = None
         if group_name is not None:
             self.group = use_group(group_name)
 
+        # module features
+        self.module_family = build_option('module_family')
+        self.module_properties = None
+        if build_option('module_properties') is not None:
+            for prop_spec in build_option('module_properties'):
+                prop_spec_parts = prop_spec.split(':')
```

Comment:
@stdweird: I'm not terribly happy with this bit. What I really need is a `listdict` type for configuration options.

Then when using `--module-properties=foo:bar;baz,one:two` would yield a value like this for `options.module_properties` (and hence also `build_option('module_properties')`):

```python
{'foo': ['bar', 'baz'], 'one': ['two']}
```